packages/fiori/test/samples/ProductSwitch.sample.html

```diff
@@ -0,0 +1,90 @@
+<header>
```

Comment:
In order to display the since tag of the component, you need to add one special marker

```html
<header class="component-heading">
	<h2 class="control-header">ShellBar</h2>
	<div class="component-heading-since">
		<span><!--since_tag_marker--></span>
	</div>
</header>
```

Inspect the ShellBar.sample.html